CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ in the naming of release branches.
 
 ### Added
 
+- Add CDDL proptests for Conway. #3226
```

Comment:
thank you for remembering the changelog! test, however, do not need mention, though, so no need to update the changelog for this PR.